packages/tao/src/shared/detect-package-manager.ts

```diff
@@ -21,28 +12,29 @@ function isPackageManagerInstalled(packageManager: string) {
   }
 }
 
-export async function detectPackageManager(
-  host: virtualFs.Host<Stats>
```

Comment:
I don't think it was necessary to use the `Host` but is there a particular reason why we don't use it anymore?

Reply:
One reason is to try to align with `detectPackageManager` of `@nrwl/workspace` which doesn't accept `Host` and the other reason is that it wasn't available on the `migrate` 🤔